provisioning/atat_provisioning_api.yaml

```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.2
+info:
+  description: >-
+    This is a DRAFT version of the ATAT Portfolio Draft API
+  version: 1.0.0
+  title: ATAT Internal API - Portfolio Drafts
```

Comment:
```suggestion
  title: ATAT Portfolio Draft API
```

Reply:
@whit1206 had asked for `Internal` to be there so I'll leave it in unless he is OK with your suggestion